src/natcap/invest/urban_nature_access.py

```diff
@@ -664,24 +882,32 @@ def _admin_level_supply_demand(
     Returns:
         ``None``
     """
-    raster_info = pygeoprocessing.get_raster_info(greenspace_budget_path)
-
-    # Reprojecting the vector here within this function allows us to produce
-    # the vector where it is used.  Otherwise, we'd be reprojecting in a
-    # separate task and then copying it to another filepath.
-    pygeoprocessing.reproject_vector(
-        admin_unit_vector_path, raster_info['projection_wkt'],
-        target_admin_unit_vector_path, driver_name='GPKG')
+    source_vector = ogr.Open(admin_unit_vector_path)
+    driver = ogr.GetDriverByName('GPKG')
+    driver.CopyDataSource(source_vector, target_admin_unit_vector_path)
+    source_vector = None
 
     target_vector = gdal.OpenEx(target_admin_unit_vector_path, gdal.GA_Update)
     target_layer = target_vector.GetLayer()
 
     supply_sum_fieldname = 'SUP_DEMadm_cap'
     undersupply_fieldname = 'Pund_adm'
     oversupply_fieldname = 'Povr_adm'
```

Comment:
Since we're making a geopackage, these field names could be more verbose, if you want

Reply:
Yes, that's true for the geopackage specification!  In this case, this is how the UG defines these fieldnames, and I'm not yet sure that folks will be happy with using a geopackage in the first place.  So for the time being, I'd like to leave them as-is, but then revisit just before the end, probably when doing the UG.  Does that sound OK?

I've added a note to self about this in https://github.com/natcap/invest/issues/722#issuecomment-1186003044

Reply:
Sounds good!